### ISSUE TITLE
9.0.0.0.rc2 popen3 doesn't work on Windows 

### ISSUE THREAD

**Issue by yamam** [jruby/jruby#3125]
```
$ jruby --version
jruby 9.0.0.0.rc2 (2.2.2) 2015-07-09 ff331eb Java HotSpot(TM) Client VM 25.45-b02 on 1.8.0_45-b14 +jit [Windows 7-x86]

$ jruby -r open3 -d -e 'Open3.popen3("notepad")'
NoMethodError: undefined method `popen3' for IO:Class
  method_missing at org/jruby/RubyBasicObject.java:1597
          popen3 at C:/jruby-9.0.0.0.rc2/lib/ruby/stdlib/jruby/open3_windows.rb:74
           <top> at -e:1
```

**Comment by headius** [comment]
Bleh. I thought I'd restored everything necessary. Will look into this.

**Comment by headius** [comment]
Ok, it's working now. I tested by forcing the use of the old Windows-friendly IO.popen3. This does not get open3 working better than in 1.7; popen2 and friends still do not work properly. We will fix this after JRuby 9.0.0.0.
